test/src/sdk-harness/test_projects/evm_ecr/mod.rs

```diff
@@ -0,0 +1,36 @@
+use fuel_core::service::Config;
+use fuel_tx::Transaction;
+use fuels_contract::script::Script;
+use fuels_signers::provider::Provider;
+use std::fs::read;
+
+async fn execute_script(bin_path: &str) -> u64 {
```

Comment:
I've used a helper like this to reduce boilerplate before too, and it makes me think we should create a small helpers lib for use in our test suite so we don't have to keep reimplementing (and maintaining) things like this in every new test. Probably something for a separate PR though.

Reply:
Yes. It would be nice to have a one-liner to execute a script and return the boolean result. We can do this directly with `forc run` so just exposing that in the SDK would be good.

Reply:
I'll mark this resolved and open an issue.